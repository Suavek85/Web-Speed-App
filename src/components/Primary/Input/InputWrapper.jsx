import React from 'react'
import classNames from 'classnames/bind'
import { useMediaQuery } from 'react-responsive'

import InputUrlWrapper from './InputUrl/InputUrlWrapper'
import InputButtonWrapper from './InputButton/InputButtonWrapper'
import styles from './Input.scss'
import { smallMobileWidth } from'../../../scss/mediaqueries'

export default function InputWrapper() {

   //STYLES
   let cx = classNames.bind(styles)
   let getClass = cx({
    block: true,
    blockSmallMobile: useMediaQuery(smallMobileWidth),
   })

  return (
    <div className={getClass}>
      <InputUrlWrapper />
      <div className={styles.btnWrapper}>
        <InputButtonWrapper />
      </div>
    </div>
  )
}