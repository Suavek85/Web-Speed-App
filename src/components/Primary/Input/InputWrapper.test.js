import React from 'react'
import { shallow } from 'enzyme'
import InputWrapper from './InputWrapper'

it('should match the snapshot', () => {
  const wrapper = shallow(<InputWrapper />)
  expect(wrapper).toMatchSnapshot()
})