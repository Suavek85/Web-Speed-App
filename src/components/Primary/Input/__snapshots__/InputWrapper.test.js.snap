// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div
  className="block"
>
  <InputUrl />
  <div>
    <InputButtonWrapper />
  </div>
  <Error />
</div>
`;
